src/components/tooltip/Tooltip.tsx

```diff
@@ -207,7 +213,7 @@ class Tooltip extends React.Component<TooltipProps, State> {
         const showTooltip = this.isShown();
 
         const withCloseButton = showCloseButton && isControlled;
-        const tetherPosition = positions[position];
+        const tetherPosition = typeof position === 'string' ? positions[position] : position;
```

Comment:
`[question-not-feedback]` is there a reason we support `string` and `array`?

Reply:
string was previously supported and I can't drop it. The custom position has 81 possibilities and I think using array of two items each has 9 possibility is better